mender-auth/cli/actions.cpp

```diff
@@ -97,7 +88,12 @@ error::Error DoAuthenticate(context::MenderContext &main_context) {
 	auto err = auth_client::FetchJWTToken(
 		client,
 		config.servers,
-		config.paths.GetKeyFile(),
+		{
+			config.security.auth_private_key == "" ? config.paths.GetKeyFile()
+												   : config.security.auth_private_key,
+			"",
```

Comment:
What about passphrase?

Reply:
Yes, glad you pointed it out.

Not sure what to do about it atm. Do we currently have any support for it? I can't find it.

This could be a good opportunity for adding it though

Reply:
It's being set [here](https://github.com/mendersoftware/mender/pull/1429/files#diff-3540d3121deba3ec9ef7262bf0093173c7717d1bfc172970cfe1a527644ac9e4R75) by command line options.

Reply:
lol, of course!